codegens/csharp-restsharp/test/unit/convert.test.js

```diff
@@ -162,6 +162,36 @@ describe('csharp restsharp function', function () {
         expect(snippet).to.include('request.AddFile("invalid src", "/path/to/file"');
       });
     });
+
+    it('should use client.UserAgent instead of AddHeader function', function () {
+      const sampleUA = 'Safari/605.1.15';
+      const expectValue = 'client.UserAgent = "Safari/605.1.15";';
```

Comment:
Combine these two `const` together. tests are failing because of that.

Reply:
Passed!